server/controllers/blogController.js

```diff
@@ -1,10 +1,51 @@
 import Blog from "../models/blog";
 
+//handle errors
+const handleErrors = (err)=>{
+    console.log(err.message, err.code)
+}
+
+
 const index =  (req,res)=>{
     Blog.find().then(result=>{res.send(result)})
     .catch(err=>{console.log(err)})
 }
+const addBlog = async (req,res)=>{
```

Comment:
Why do you have changes for adding/updating and deleting blog posts in a retrieve all queries PR?

Reply:
Here i think i forgot to pull first from develop